apis/taskcluster.py

```diff
@@ -44,11 +44,11 @@ def from_string(s, logger):
             return Classification.NewFailure
         elif s == "new failure not classified":  # from TC
             return Classification.NewFailure
-        elif s == "fixedByCommit":  # from Push Health
+        elif s in ["fixedByCommit", "intermittent needs bugid"]:  # from Push Health
             return Classification.NotYourFault
         elif s in ["autoclassified intermittent", "expected fail", "fixed by commit", "infra"]:  # from TC
             return Classification.NotYourFault
```

Comment:
This string is from TC, right?

```suggestion
        elif s in ["fixedByCommit"]:  # from Push Health
            return Classification.NotYourFault
        elif s in ["autoclassified intermittent", "expected fail", "fixed by commit", "infra", "intermittent needs bugid"]:  # from TC
            return Classification.NotYourFault
```

Reply:
No, it's from Push Health: https://github.com/mozilla-services/updatebot/issues/399